pkg/dwarf/line/state_machine.go

```diff
@@ -320,6 +298,40 @@ func (lineInfo *DebugLineInfo) LineToPC(filename string, lineno int) uint64 {
 	return fallbackPC
 }
 
+// LineToPCIn returns the first PC for filename:lineno in the interval [startPC, endPC).
+// basePC will be used for caching, it's normally the entry point for the
+// function containing pc.
```

Comment:
Let's add just a little bit more info here, specifically mention something about how we can't rely simply on the file:line and instead must provide an address range because of inlined functions.